### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+project: zstd-rs
+target: zstd-rs
+
+image: ghcr.io/ebell495/zstd-rs:latest
+duration: 300
+
+cmds:
```
Comment (xansec) [inline Mayhemfile:7]: There should only be one target per Mayhemfile. Please either omit one target or split into two Mayhemfiles (as described here: https://community.forallsecure.com/t/error-the-mayhem-for-code-scan-detected-the-mayhem-run-for-your-target-was-unsuccessful-and-multiple-docker-files-in-the-repository/350/5)

**Review by xansec — CHANGES_REQUESTED** [review]

File: Mayhemfile (hunk 1 above)
Comment (xansec) [reply Mayhemfile:7]: Also make sure that the target name reflects the command being run.

**Comment by ebell495** [comment]
I've separated the targets and fixed the names.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**